riscv/src/runtime.rs

```diff
@@ -148,6 +148,48 @@ impl Runtime {
         self.syscalls.contains_key(&s)
     }
 
+    pub fn with_keccak(mut self) -> Self {
+        self.add_submachine(
+            "test_data::asm::keccakf::KeccakF",
```

Comment:
I don't think this is accessible, you need to add a new machine to `std`.

Reply:
It currently resides in the test folder only till the machine is complete, but I agree that eventually it'll be in `std`.

Reply:
is this path accessible though?

Reply:
Not super sure but I'll test it out.